modules/RTC/TPCUtils.js

```diff
@@ -220,26 +220,22 @@ export class TPCUtils {
             this.pc.localTracks.set(localTrack.rtcId, localTrack);
             transceiver.direction = 'sendrecv';
 
-            return true;
+            return Promise.resolve(false);
```

Comment:
is `this.pc.peerconnection.addStream(localTrack.getOriginalStream());` async? does it require renegotiation?

Reply:
no addStream is not async. This doesn't require renegotiation. If renegotiation is required, browser will fire a negotiationneeded event which would trigger a O/A.